index.js

```diff
@@ -5,10 +5,14 @@ let upstreamTransformer = null;
 
 const reactNativeVersionString = require('react-native/package.json').version;
 
+const metroVersion = parseFloat(require('metro/package.json').version);
```

Comment:
Parsing the version as a float seems a little hacky to me.
Could you use the same syntax as `reactNativeMinorVersion` ?

Like:
```js
const metroVersionString = require('metro/package.json').version;

const metroMinorVersion = semver(metroVersionString).minor;
```